WATOLINK/bci-gaming
# Grace - Animations & Animation Movement Integration

## ClickUp ticket link

[Game - Animations](https://app.clickup.com/t/860qb73r6)
[Game - Animation Movement Integration](https://app.clickup.com/t/8684rv0td)



## Implementation description
Completed all player animations (idle, walk, takeoff, jump, land, melee, ranged, hurt, death). Completed all bandit, boss cactus, and cowboy animations. Combined movement with appropriate animations.



## Steps to test
1. Click play and press on the left & right arrows on the keyboard. Ensure the player moves in the correct direction, and after releasing the keyboard, the player faces the same direction they moved toward.
2. Press the up button. The player should jump. Pressing left/right with the up button will make the player jump toward their left/right.
3. Make sure animations are looping (i.e. the sprite should not freeze)
4. Find all animations in the Animations folder under Assets. Find all spritesheets under Sprites folder under Assets.



## What should reviewers focus on?
Fluency of animation, character movement


## Checklist
